src/dicomweb_client/cli.py

```diff
@@ -58,10 +58,18 @@ def _get_parser():
         '--cert', dest='cert', metavar='CERT-FILE',
         help='path to a client certificate file in PEM format'
     )
+    parser.add_argument(
+        '-t', '--token', dest='token', metavar='TOKEN',
+        help='bearer token for authentication with the DICOMweb service'
+    )
     parser.add_argument(
         '--url', dest='url', metavar='URL',
         help='uniform resource locator of the DICOMweb service'
     )
+    parser.add_argument(
+        '--chunk_size', dest='chunk_size', metavar='CHUNK_SIZE',
```

Comment:
Please change the value of metavar from ``CHUNK_SIZE`` to ``NUM`` to follow the same logic of the other arguments of the command line interface.